task/mongo-tasks.js

```diff
@@ -80,15 +109,49 @@ async function task_1_3(db) {
  *
  */
 async function task_1_4(db) {
-    throw new Error("Not implemented");
+    const result = await db.collection('orders').aggregate([
+        {
```

Comment:
Works... but in 2 queries it can work much faster. `$group` and `$lookup` are quite slow in comparison with simple `count`